### PR TITLE
Fix - Missing Utm interface properties

### PR DESCRIPTION
**Pull request by marko-ogg**
Oversight from this PR (after moving `Utm.ts` to `common`): https://github.com/prezly/javascript-sdk/pull/87

### CODE REVIEW

File: src/types/common/index.ts [hunk 1]
```diff
@@ -2,4 +2,5 @@ export * from './Notification';
 export * from './OEmbedInfo';
 export * from './Pagination';
 export * from './Query';
+export * from './Utm';
```
Comment (e1himself) [inline src/types/common/index.ts:5]: Actually in #87 I've moved it to Subscriptions SDK, as it was a local type anyway (not publicly exported from `types` folder).

https://github.com/prezly/javascript-sdk/blob/269a98f3a1ca10e0b621f0e99f9767401689b0ee/src/Sdk/Subscriptions/types.ts#L3-L9

So please remove the dupe definition of it in `Subscriptions/types.ts`.

**Review by e1himself — APPROVED** [review]

File: src/types/common/index.ts (hunk 1 above)
Comment (marko-ogg) [reply src/types/common/index.ts:5]: But you also left empty `Utm.ts` file in `common`: https://github.com/prezly/javascript-sdk/blob/269a98f3a1ca10e0b621f0e99f9767401689b0ee/src/types/common/Utm.ts

Comment (e1himself) [reply src/types/common/index.ts:5]: :open_mouth:

**Pull request merged**